tests/Jira/Api/Client/AbstractClientTestCase.php

```diff
@@ -185,6 +185,11 @@ public function emptyResponseWithKnownHttpCodeDataProvider()
 		);
 	}
 
+	public function testErrorResponseValueReturned()
+	{
+		$this->traceRequest(Api::REQUEST_GET, array('http_code' => '403', 'response_mode' => 'trace'));
```

Comment:
That doesn't look to explicit. I expect some predetermined output being sent back from REST endpoint and then assertion made on actual output.